modules/checkout/src/Plugin/Commerce/CheckoutFlow/CheckoutFlowBase.php

```diff
@@ -156,7 +174,7 @@ public function getSteps() {
     return [
       'payment' => [
         'label' => $this->t('Payment'),
-        'next_label' => $this->t('Pay and complete purchase'),
```

Comment:
Didn't we say that "Pay and complete purchase" makes sense for both offsite and onsite? I find it more descriptive than just "Complete purchase"